app/assets/javascripts/bind_event_to_preview_page.js

```diff
@@ -26,7 +46,13 @@ $(document).ready(function () {
     var title = $('#event_title').val() === '' ? 'Event title goes here' : $('#event_title').val();
     $('.our-event-title').html(title);
     $('.our_event_description').html(description);
-    $('.our-event-date').html(start_date + ' to ' + end_date);
+    if (rec_text !== ''){
+      $('.our-event-date').html(start_date + ' - ' + end_date);
+      $('.our-event-time').html(rec_text + start_time + ' to ' + end_time);
+    } else {
+      $('.our-event-date').html(start_date + ', ' + start_time + ' - ' + end_date + ', ' + end_time);
+      $('.our-event-time').html('');
```

Comment:
NITPICK: `$('.our-event-date')` and `$('.our-event-time')` can be assigned to variables for multiple usage.